README.md

```diff
@@ -9,13 +9,14 @@ The main entrypoint (or homepage) is `index.md` and new pages must be referenced
 
 ### To test locally
 
-Making sure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Bundler](https://bundler.io/) are installed in your machine, run the following:
+Making sure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) `2.7` and [Bundler](https://bundler.io/) are installed in your machine, run the following:
 
 ```sh
 git clone git@github.com:ajuna-network/ajuna-network.github.io.git
 cd ajuna-network.github.io
 
 bundle install
+bundle config set --local path 'vendor/bundle'
```

Comment:
Don't want bundle polluting our environments with ruby stuff :D

Reply:
Yeah it's better to install locally :) nice catch